.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "MIEngine Codespace",
+	"settings": {
+		"workbench.colorTheme": "Default Dark+",
+		"terminal.integrated.defaultProfile.linux": "pwsh"
+	},
+	"extensions": [
+		// VS Code specific
+		"coenraads.bracket-pair-colorizer",
+		"vscode-icons-team.vscode-icons",
+		"editorconfig.editorconfig",
+		// GitHub specific
+		"eamodio.gitlens",
+		"cschleiden.vscode-github-actions",
+		"redhat.vscode-yaml",
```

Comment:
This repo uses GitHub YAML and AzurePipeline YAML.

The better extension to use is https://github.com/Microsoft/azure-pipelines-vscode